proto/cosmos/bank/v1beta1/cli.yaml

```
@@ -0,0 +1,20 @@
+tx:
+  service: cosmos.bank.v1beta1.Msg
+  method_options:
+    Send:
+      field_options:
+        from_address:
+          position: 0
+        to_address:
+          position: 1
+        amount:
+          position: 2
```

Really nice! Perhaps a different syntax is what I'd first pass think, but I'm happy either way

```suggestion
send:
  proto_msg: cosmos.bank.v1beta1.MsgSend
  short_example: send {from} {to} {amount}
  long_example: send osmo1myAddr... osmo1toAddr... 1osmo
  args:
    from:
      proto_field: "from_address"
    to:
      proto_field: 1 # highlights that we can mix field number or strings here
    coins:
      multi_arg_parse: # highlights that can span across multiple args
      proto_field: 2
```

---

Looking at this, theres not realy any difference in CLI specification, beyond how the top level looks